infrastructure/account-data-deleter/src/dataDeleterApp.ts

```diff
@@ -94,10 +94,14 @@ export class DataDeleterApp extends Construct {
           threshold: 25,
         },
       },
+      taskSize: {
+        cpu: config.isDev ? 512 : 2048,
+        memory: config.isDev ? 2048 : 16384,
+      },
```

Comment:
AccountDataDeleter appears to have a memory leak, because its memory usage climbs to 100% over the course of a few weeks. This may be problematic if the memory leak scales with the number of requests, instead of only over time. Giving it more memory should at least make restarts less frequent. 🤞

![image](https://github.com/user-attachments/assets/4dc971de-c189-4f06-9bd7-e68faf18b3a9)